base-notebook/test/test_container_options.py

```diff
@@ -187,3 +187,28 @@ def test_group_add(container, tmpdir):
     assert rv == 0 or rv["StatusCode"] == 0
     logs = c.logs(stdout=True).decode("utf-8")
     assert "uid=1010 gid=1010 groups=1010,100(users)" in logs
+
+
+def test_container_not_delete_bind_mount(container, tmp_path):
+    """Container should not delete host system files when using the (docker)
+    -v bind mount flag and mapping to /home/jovyan.
+    """
+    d = tmp_path / "data"
+    d.mkdir()
+    p = d / "foo.txt"
+    p.write_text("some-content")
+
+    container.run(
+        tty=True,
+        user="root",
+        working_dir="/home/",
+        environment=[
+            "NB_USER=user",
+            "CHOWN_HOME=yes",
+        ],
+        volumes={d: {"bind": "/home/jovyan/data", "mode": "rw"}},
+        command=["start.sh", "jupyter", "lab"],
+    )
+    time.sleep(2)
```

Comment:
Could you please check rv here as it was done in test_group_add test?
```
c = container.run(...)
rv = c.wait(timeout=5)
assert rv == 0 or rv["StatusCode"] == 0
```

Reply:
Added 👍

Reply:
Thanks 👍